frontend/app/tests/e2e/pages/assets-manager-page.ts

```diff
@@ -24,11 +24,14 @@ export class AssetsManagerPage {
     cy.get('[data-cy=asset_table_filter]').type(
       `{selectall}{backspace}symbol: ${asset}{enter}{esc}`
     );
+    cy.get('.v-data-table__empty-wrapper', {
+      timeout: 300000
+    }).should('not.exist');
+    cy.get('.v-data-table__progress', { timeout: 300000 }).should('not.exist');
```

Comment:
instead of waiting for `x` it is better to verify that the UI finished loading and doesn't display the empty wrapper.